src/api/entityResolution/index.ts

```diff
@@ -136,4 +138,27 @@ export class EntityResolutionAPI extends Request {
       params: params
     });
   }
+
+  /**
+   * Get informations about the entity resolution license, across all data-sources.
+   */
+  getLicenseInfo(this: Request<EntityResolutionLicenseInfo>) {
+    return this.request({
+      errors: [UNAUTHORIZED, FORBIDDEN],
+      url: '/entityResolution/license',
+      method: 'GET'
+    });
+  }
```

Comment:
We may later consider adding a `POST /entityResolution/license` endpoint, to upload a license.